.gitignore

```diff
@@ -1,37 +1,92 @@
-# Logs
-logs
-*.log
-
-# Runtime data
-pids
-*.pid
-*.seed
-
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
-
```

Comment:
I don't like "could be useful" entries because I feel that "could be useful" code should not exist in a project. In a project's dependencies—yes. Oh, and I do consider this code, even though it's not programming language code.

And if anyone's environment/tooling/IDE/OS generates any artifact files then they should create their own `.git/info/exclude` or `$HOME/.config/git/ignore` and ignore those files there and not in the project. That's how I feel. For more information, read [the docs](https://git-scm.com/docs/gitignore).